oneandone/src/main/java/org/apache/jclouds/oneandone/rest/OneAndOneApi.java

```diff
@@ -16,13 +16,11 @@
  */
 package org.apache.jclouds.oneandone.rest;
 
-import autovalue.shaded.com.google.common.common.annotations.Beta;
 import java.io.Closeable;
 import org.apache.jclouds.oneandone.rest.features.ImageApi;
 import org.apache.jclouds.oneandone.rest.features.ServerApi;
 import org.jclouds.rest.annotations.Delegate;
 
-@Beta
```

Comment:
You can keep this, but taking care of using the right import. Just `com.google.common.common.annotations.Beta`.